### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by tthyer**
Adding code of conduct based on the recommended one from Github.
Please read and indicate whether this is good, should be edited, or other concerns.
Thinking of recommending that we always add a code of conduct in our repositories.

### CODE REVIEW

**Comment by tthyer** [comment]
@thomasyu888 pointed out in chat that the Synapse python client already has a [code of conduct](https://github.com/Sage-Bionetworks/synapsePythonClient/blob/develop/CODE_OF_CONDUCT.md). It is based on the same template, just an earlier version. It seems that ideally Sage adopts one code of conduct that we could link to from our repositories. Thoughts?

**Review by philerooski — APPROVED** [review]
Agree. Sage should have a standard code of conduct across all its software projects.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+dpe@sagebase.org.
```
Comment (BrunoGrandePhD) [inline CODE_OF_CONDUCT.md:63]: Do we want to replicate the paragraph on enforcement from our organizational code of conduct, which includes an anonymous hotline? If not, are we sure DPE should be the recipient here? Why not HR?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```
[the remark this answers is not in the document]
Comment (BrunoGrandePhD) [reply CODE_OF_CONDUCT.md:1]: Another option is to update our organizational code of conduct to use version 2.0 and link to what.

[the remark this answers is not in the document]
Comment (tthyer) [reply CODE_OF_CONDUCT.md:1]: @BrunoGrandePhD I agree that is optimal. I'll figure out who to contact about doing those things.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (tthyer) [reply CODE_OF_CONDUCT.md:63]: If we link out to the Sage Code of Conduct, then this is not a worry

File: CODE_OF_CONDUCT.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (tthyer) [reply CODE_OF_CONDUCT.md:1]: And I see you already contacted them -- thanks!

**Pull request merged**